.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,412 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <option name="LINE_SEPARATOR" value="&#10;" />
+    <option name="RIGHT_MARGIN" value="150" />
```

Comment:
150 is too much IMHO and Google Java Style Guide talks about [100 as column limit](https://google.github.io/styleguide/javaguide.html#s4.4-column-limit) and we had 120 before so I would keep it in 120. I know we already changed this in the library but I will probably set it again to 120.

Reply:
you will generate more line breaks, even though you see space right to line

Reply:
Generating more line breaks is not something bad most of the times, it makes the code more readable.

Reply:
120 is fine when reading changes on GH PRs as well. Can you set it to 120 for now?

Reply:
Ok, I will do. But I wait, till a final merge can be done within minutes or hours, because I will start a **final** reformat after this. And to be honest, I've no motivation to do it again and again ...
Please have a look on how much conflicts I got after 12 days !

Reply:
> 120 is fine when reading changes on GH PRs as well. Can you set it to 120 for now?

[Here](https://github.com/owncloud/android/pull/2676) i apply it